### ISSUE TITLE
Large array support - totalItemCount 2000000

### ISSUE THREAD

**Issue by davidterins** [petyosi/react-virtuoso#86]
Hi, 

Im currently using react-virtualized for a project where I have over 2 million items to display, however managing dynamic row-height and positions of elements can sometimes be a bit off.

So I though react-virtuoso seemed like an interesting alternative. When doing so I quickly ran in to the problem that react-virtuoso will not work for much more than 226710 items when trying it out at https://stackblitz.com/run?file=example.js, whereas react-virtualized seem to have no limits https://bvaughn.github.io/react-virtualized/#/components/List. Are there any plans of increasing the total item count for react-virtuoso, if possible, that would be really awesome!

**Comment by petyosi** [comment]
Hey, 

@toonvd was interested in contributing this, but I am not sure if he got to it. I may check and do this in the upcoming confinement days.

**Comment by toonvd** [comment]
Hi

I started it but haven't gotten a chance to finish since I moved to other projects at work.
I finished reading your articles and have taken the first steps to determine the max height for the current browser. I first went for the UserAgent but as some people manipulate that, I don't think that would be a good way to go. Unless we could determine a fallback (bare minimum). Maybe we should just go for Chromium based vs none Chromium based as Chromium based browsers have the same limitations? Edge for example also has the window.chrome object.
